apps/server/src/routes/swagger/submission-api.yml

```diff
@@ -1,8 +1,33 @@
 # Description of Submission API
 
+/submission/{submissionId}:
+  get:
+    summary: Get active submission
+    tags:
+      - submission
+    parameters:
+      - name: submissionId
+        in: path
+        type: string
+        required: true
+    responses:
+      200:
+        description: Active Submission
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ActiveSubmissionResult'
+
+      401:
+        $ref: '#/components/responses/UnauthorizedError'
+      500:
+        $ref: '#/components/responses/ServerError'
+      503:
+        $ref: '#/components/responses/ServiceUnavailableError'
+
 /submission/category/{categoryId}:
   get:
-    summary: Get active submission for a category
+    summary: Get active submissions for a category
```

Comment:
Perhaps this is better described as `List active submissions for a category` since it doesn't get the entire active submission.

Reply:
Changes as suggested. Thanks.